census/controllers/census.js

```diff
@@ -219,6 +221,39 @@ var submit = function(req, res) {
   });
 };
 
+let _getDiscussionURL = function(req, dataset, place) {
+  /*
+    If `submission_discussion_url` is defined in settings and it is in the
+    format: https://discuss.okfn.org/c/<topic>/<subtopic>, return a new topic
+    url with a prepopulated topic for place and dataset. Otherwise, return the
+    original `submission_discussion_url` without modification. If
+    `submission_discussion_url` is undefined return an empty string.
+  */
+  let submissionDiscussionURL =
+    _.get(req.params.site.settings, 'submission_discussion_url', '');
+  let parsedURL = url.parse(submissionDiscussionURL);
+  // URL is a discourse link
+  if (parsedURL.hostname === 'discuss.okfn.org') {
+    let splitPathName = _.trimLeft(parsedURL.pathname, '/').split('/');
+    // URL is a category link
+    if (splitPathName[0] === 'c') {
```

Comment:
I just wanted to check this is a category path (`/c/whatever`) for Discourse, as we need the specific categories to do the transformation correctly. If it's a `discuss.okfn.org` link, but not a category path, the transformation is skipped, and the `submissionDiscussionURL` is passed through the function.